### ISSUE TITLE
[Effect Suggestion] Online Experience

### ISSUE THREAD

**Issue by DaBestIvo** [gta-chaos-mod/ChaosModV#2445]
**Name of effect**
Online Experience

**Duration (choose one and replace the corresponding "[ ]" with "[X]")**
- [ ] Long
- [x] Short
- [] No duration

**Effect Description**
When the effect is chosen, the player first hears an explosion above them before 3 or 4 exploded oppressors rain down. It would also be cool if the effect spawned a homing missile aimed at the player once or twice at a reasonable distance.
Maybe a bit too much, but also a money drop.

**Comment by DaBestIvo** [comment]
I know things like this have been suggested already, but I feel like this is more balanced.

**Comment by VDKA-Deado** [comment]
I agree that this seems much more balanced, and is pretty much the online experience in a nutshell. Though I do indeed think (personally) that the money drop is a bit unnecessary, especially since that effect already exists.

**Comment by DaBestIvo** [comment]
Yeah, I agree. Maybe just explosions in the distance. From: VDKA-DeadoSent: Monday, September 27, 2021 7:47 PMTo: gta-chaos-mod/ChaosModVCc: DaBestIvo; AuthorSubject: Re: [gta-chaos-mod/ChaosModV] [Effect Suggestion] Online Experience (#2445) I agree that this seems much more balanced, and is pretty much the online experience in a nutshell. Though I do indeed think (personally) that the money drop is a bit unnecessary, especially since that effect already exists.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
